src/form.js

```diff
@@ -177,8 +173,7 @@ define([
 
     Form.prototype = {
         getBasePath: function () {
-            // the choice to use dataTree instead of controlTree is arbitrary
```

Comment:
Heh, when I wrote this comment I only meant it within the context of this function. Kind of amazing that you were able to run with it and figured out a way to apply it to all of Vellum.